modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/GeneTreeHighlighting_conf.pm

```diff
@@ -45,6 +45,10 @@ sub default_options {
     run_all      => 0,
     antispecies  => [],
     meta_filters => {},
+    production_db => 'ensembl_production',
+    production_host => 'mysql-ens-meta-prod-1-ensprod',
+    compara_db => '',
```

Comment:
I think it'd be better to delete this line. If there is no default for compara_db, then if you forget to supply the parameter, it will fail at the init_pipeline step. Otherwise, the first analysis will fail, because the command will not work if an empty string is used.

Reply:
Well spotted, thanks @james-monkeyshines. I've removed the line now.